eng/DownloadSharedSource.ps1

```diff
@@ -19,8 +19,8 @@ Write-Host "git sparse-checkout init"
 git sparse-checkout init
 Write-Host "git sparse-checkout set --no-cone $azCoreSharedPath $armCoreSharedPath"
 git sparse-checkout set --no-cone $azCoreSharedPath $armCoreSharedPath
-Write-Host "git checkout"
-git checkout
+Write-Host "git checkout mnash-updatePageableHelpers"
```

Comment:
By accident?

Reply:
No this was on purpose because I needed a non merged version but I have updated it back to main since that was merged into main here https://github.com/Azure/azure-sdk-for-net/pull/38399